suite-native/module-check-backup/src/screens/DeviceCheckBackupUnsupportedModelScreen.tsx

```diff
@@ -0,0 +1,91 @@
+import { RouteProp, useNavigation, useRoute } from '@react-navigation/native';
+
+import { Box, Button, IconListTextItem, TitleHeader, VStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { useOpenLink } from '@suite-native/link';
+import {
+    DeviceCheckBackupStackParamList,
+    DeviceCheckBackupStackRoutes,
+    DeviceOnboardingStackParamList,
+    DeviceOnboardingStackRoutes,
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes,
+    Screen,
+    ScreenHeader,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+import { SUITE_WEB_DEVICE_SETTINGS_URL } from '@trezor/urls';
+
+type RouteProps = RouteProp<
+    DeviceCheckBackupStackParamList,
+    DeviceCheckBackupStackRoutes.UnsupportedModel
+>;
+
+type NavigationProps = StackToStackCompositeNavigationProps<
+    DeviceOnboardingStackParamList,
+    DeviceOnboardingStackRoutes.RecoveryUnsupported,
+    DeviceSettingsStackParamList
+>;
+
+export const DeviceCheckBackupUnsupportedModelScreen = () => {
+    const { params } = useRoute<RouteProps>();
+
+    const navigation = useNavigation<NavigationProps>();
+    const openLink = useOpenLink();
+
+    const redirectToWeb = () => {
+        openLink(SUITE_WEB_DEVICE_SETTINGS_URL);
+    };
+
+    const redirectHome = () => {
+        navigation.navigate(DeviceSettingsStackRoutes.DeviceSettings);
+    };
```

Comment:
Nitpick for naming - `redirectToDeviceSettings`. You could also use `navigateToInitialScreen` in case this would ever be used elsewhere

Reply:
https://github.com/trezor/trezor-suite/pull/19963/commits/1b1a1219aca94a82c10fa5da8185455069f85627

Reply:
@BrantalikP Are you sure? That's weird. It should work that it will move you to previous stack. Something might be off...

If there is more than one route in the stack state, it will pop to top and then go back. If only one route, it will go back (so go to previous stack). 

But it's not critical, right now it is not necessary to go to this screen from other flows.

Reply:
Hmm, I think so, because I remembered that you actually suggested it to Peta somewhere, so I wanted to give it a shot,  but unfortunately, it didn’t work as expected. Let's investigate it later. anyway. Thanks 💪